tests/test_checks.py

```diff
@@ -93,3 +94,22 @@ def test_enum_type_boolean(tmpdir, rc):
                              |         | false
     ''')
     commands.check(context, manifest)
+
+
+def test_filename(tmpdir, rc):
```

Comment:
```suggestion
def test_filename(tmp_path: Path, rc: RawConfig):
```

Always tuse `tmp_path`, because `tmpdir` is becoming deprecated. And, please always use type annotations.